geocruncher/ComputeIntersections.py

```diff
@@ -25,7 +25,10 @@ def computeRank(a,b):
             evaluate_z = topography.evaluate_z
             ranks=model.rank
             return ranks([a,b,evaluate_z([a,b])])
-
+            
+        def findMin(a,b):
+            return np.minimum(a,b)

```

Comment:
Is this method really useful? Why not just call ``np.minimum`` directly where it is used?